### PR TITLE
Wire sky_shell.mojo to draw via MGL

### PR DESCRIPTION
**Pull request by abarth**
After this patch, we can execute Dart code and draw the result via MGL. We
still need to plumb in services.

### CODE REVIEW

**Comment by abarth** [comment]
@jamesr @eseidelGoogle

**Comment by eseidelGoogle** [comment]
lgtm!

File: sky/shell/BUILD.gn [hunk 1]
```diff
@@ -240,13 +267,15 @@ if (is_android) {
...
 
-    deps = common_deps + [
-             ":common",
-             "//mojo/edk/system",
-             "//mojo/common",
-           ]
+    deps = [
+      "//mojo/common",
+      "//mojo/edk/system",
+      ":common",
```
Comment (jamesr) [inline sky/shell/BUILD.gn]: put local references before ones from other files (i.e. sort ":" before "//")

File: sky/shell/rasterizer.h [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+#include <memory>
+
+#include "base/callback.h"
+#include "base/memory/scoped_ptr.h"
+#include "sky/compositor/layer_tree.h"
+
+namespace sky {
+namespace shell {
+
+typedef base::Callback<void(scoped_ptr<compositor::LayerTree>)> RasterCallback;
```
Comment (jamesr) [inline sky/shell/rasterizer.h]: the version with 'using' reads better, imo

File: sky/shell/BUILD.gn (hunk 1 above)
Comment (abarth) [inline sky/shell/BUILD.gn]: Ah, sorry.

File: sky/shell/rasterizer.h (hunk 2 above)
Comment (abarth) [inline sky/shell/rasterizer.h]: Ok.  Will do in the future.

**Pull request merged**